### PR TITLE
refine sdn migration to ovn jobs

### PR DESCRIPTION
**Pull request by zhaozhanqi**
for sdn migration ovn profile. 

1. remove all workflow like `sdn2ovn`. eg. `cucushift-installer-rehearse-azure-ipi-ovn-sdn2ovn` and only keep ipi aws platform to make sure ovn regression related features are working well after migration. 
2. just using chain `cucushift-sdn-migration-ovn` and cover aws/azure/gcp/vsphere platform and other platforms if the base workflow exist. 

eg. 
```
- as: vsphere-ipi-sdn-migration-ovn-p2-f7
  cron: 11 21 2,10,17,24 * *
  steps:
    cluster_profile: vsphere
    env:
      UPGRADE_CHECK_RUN_TAGS: '@4.13 and @vsphere-ipi and @upgrade-check and @network-ovnkubernetes
        and not @fips'
      UPGRADE_PRE_RUN_TAGS: '@4.13 and @vsphere-ipi and @upgrade-prepare and @network-openshiftsdn
        and not @fips'
    test:
    - chain: cucushift-sdn-migration-ovn
    workflow: cucushift-installer-rehearse-vsphere-ipi-sdn

@openshift/team-sdn-qe 

cc @jiajliu @gpei  @liangxia

### CODE REVIEW

**Comment by weliang1** [comment]
@zhaozhanqi Question about your statement "remove all workflow like sdn2ovn...........only keep ipi aws platform to make sure ovn regression related features are working well after migration". Will ipi-aws continue use sdn2ovn workflow not chain cucushift-sdn-migration-ovn?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -1997,16 +2009,6 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnected-secureboot
-- as: vsphere-upi-ovn-sdn2ovn-p2-f7
```
Comment (weliang1) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2000]: Need also remove `as: azure-ipi-ovn-sdn2ovn-p2-f7`

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml [hunk 2]
```diff
@@ -15894,12 +15894,12 @@ periodics:
     repo: openshift-tests-private
   labels:
     ci-operator.openshift.io/cloud: vsphere
-    ci-operator.openshift.io/cloud-cluster-profile: vsphere
+    ci-operator.openshift.io/cloud-cluster-profile: vsphere-discon
     ci-operator.openshift.io/variant: amd64-nightly
     ci.openshift.io/generator: prowgen
     job-release: "4.13"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-vsphere-upi-ovn-sdn2ovn-p2-f7
```
Comment (weliang1) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:15902]: Should also remove ` periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-azure-ipi-ovn-sdn2ovn-p2-f7`?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml (hunk 1 above)
Comment (zhaozhanqi) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:2000]: yes, good catch . done

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml (hunk 2 above)
Comment (zhaozhanqi) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:15902]: fixed.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 3]
```diff
@@ -1963,6 +1952,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-sdn
+- as: vsphere-ipi-sdn-migration-ovn-p2-f7
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1955]: Also need to remove `vsphere-upi-ovn-sdn2ovn`, right?

Comment (zhaozhanqi) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1955]: yes,  we have `vsphere-ipi-sdn-migration-ovn-p2-f7` can cover vsphere

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1955]: To get those unneeded jobs removed in this PR to take effect, we can let this PR go, and use a follow up PR to remove `vsphere-upi-ovn-sdn2ovn` if that's also unneeded.

**Pull request merged**